pages/quantum.js

```diff
@@ -53,139 +61,453 @@ import {
+            <h6 class="TextFormattingH6">
+              Tooltips provide additional information upon hover or focus. They
+              often contain helper text that is contextual to an element.
+            </h6>
+            <Tooltip placement="top" text="Example Tooltip Text">
+              Pass the pointer over me
+            </Tooltip>
+          </div>
+        </div>
+      </div>
+      <div class="ColoredBackground ComponentsContainer">
+        <h4 class="TextFormattingH4">Carousel</h4>
+        <h6 class="TextFormattingH6">
+          Carousel is a component that align cards with a series of content.
+          Works with images, texts, or custom markups. Also includes support for
+          indicators and previous/next controls.
+        </h6>
+        <Carousel cards={cardsMock} />
+      </div>
+      <div class="FlexContainer ColoredBackground PaddingLeftRight MarginTopBottom">
+        <div class="Width  ComponentsContainer WhiteBackground RoundedEdges">
+          <h4 class="TextFormattingH4">Social Button</h4>
+          <h6 class="TextFormattingH6">
+            These buttons are used to run actions related to their respective
+            providers. Example: login with OAuth API from Google.
+          </h6>
+          <SocialButton provider="facebook" />
+        </div>
+        <div class="Width  ComponentsContainer WhiteBackground RoundedEdges">
+          <h4 class="TextFormattingH4">Info Icon</h4>
+          <Icon name="info" />
+        </div>
+        <div class="Width  ComponentsContainer WhiteBackground RoundedEdges">
+          <h4 class="TextFormattingH4">Avatar</h4>
+          <h6 class="TextFormattingH6">
+            Avatars are used to display a thumbnail of the user's profile
+            picture or an user icon if the picture is not defined. It can also
+            display a dot to alert that there is some notification.
+          </h6>
+          <Avatar />
+        </div>
+        <div class="Width   ComponentsContainer WhiteBackground RoundedEdges">
+          <h4 class="TextFormattingH4">Socials</h4>
+          <h6 class="TextFormattingH6">
+            Socials show the icons of respective Social Medias{" "}
+          </h6>
+          <Socials items={socialMocks} />
+        </div>
+      </div>
+      <div class="FlexContainer ColoredBackground PaddingLeftRight MarginTopBottom Overflow ">
+        <div class="Width  ComponentsContainer WhiteBackground RoundedEdges">
+          <h4 class="TextFormattingH4">Checkbox Group</h4>
+          <h6 class="TextFormattingH6">
+            Gets values from a single onChange callback prop and apply a single
+            error message to a group of checkboxes.
+          </h6>
+          <CheckboxGroup
+            options={[{ checked: false, label: "Test", name: "Test" }]}
+          />
+        </div>
+        <div class="Width  ComponentsContainer WhiteBackground RoundedEdges">
+          <h4 class="TextFormattingH4">Radio Group</h4>
```

Comment:
how of the example used in storybook to use here?
https://catho.github.io/quantum/?path=/docs/forms-radiogroup--default
a example more complete, to any options.